documentation/modules/proc-deploying-service-broker.adoc

```diff
@@ -1,119 +1,41 @@
 // Module included in the following assemblies:
 //
 // assembly-installing-manual-steps.adoc
-
+ifeval::["{cmdcli}" == "oc"]
 [id='deploying-service-broker-{context}']
 = (Optional) Deploying the service broker
 
-The service broker provides an implementation of the Open Service Broker API that integrates with the Kubernetes Service Catalog. The service broker requires the standard authentication service to be deployed.
+The service broker provides an implementation of the Open Service Broker API that integrates with the {PlatformName} Service Catalog. The service broker requires the standard authentication service to be deployed.
 
-ifeval::["{PlatformName}" == "OpenShift"]
 .Prerequisites
-
 * The service broker requires the standard authentication service to be deployed.
+ifeval::["{PlatformName}" == "OpenShift"]
 * To install the {ProductName} Service Broker on {PlatformName}, you must have `cluster-admin` access to set up the
 required roles for delegating authentication to the {PlatformName} master; otherwise, you are restricted
 to using templates for creating address spaces. For more information about how to deploy without `cluster-admin` access,
 which restricts {ProductName} to using templates, see xref:installing-without-cluster-admin-privileges-{context}[].
 endif::[]
-
-.Procedure
-
-. Create a service account for the {ProductName} service broker:
-+
-[options="nowrap",subs="attributes"]
-----
-{cmdcli} create sa enmasse-admin
-----
-
-ifeval::["{cmdcli}" == "oc"]
-. Create cluster-wide roles used by the `enmasse-admin` service account:
-+
-[options="nowrap"]
-----
-oc login -u system:admin
-oc create -f ./resources/cluster-roles/service-broker.yaml
-----
-
-. Grant privileges to the service account:
-+
-[options="nowrap"]
-----
-oc login -u system:admin
-oc policy add-role-to-user admin system:serviceaccount:enmasse:enmasse-admin
-oc adm policy add-cluster-role-to-user enmasse.io:service-broker system:serviceaccount:enmasse:enmasse-admin
-oc adm policy add-cluster-role-to-user system:auth-delegator system:serviceaccount:enmasse:enmasse-admin
-----
-+
-NOTE: You can log in again as the regular user after this step.
-endif::[]
-
-. Create a certificate to use for the service broker. For testing purposes, you can create a self-signed certificate:
+* Modify the installation files according to the namespace the Service Broker is going to be installed in
```

Comment:
Modify the installation files to reflect the namespace in which the Service Broker will be installed: